app/src/main/java/net/gsantner/markor/format/general/DatetimeFormatDialog.java

```diff
@@ -94,8 +94,6 @@ public static void showDatetimeFormatDialog(final Activity activity, final Highl
         final Button datePickButton = viewRoot.findViewById(R.id.start_datepicker_button);
         final Button timePickButton = viewRoot.findViewById(R.id.start_timepicker_button);
         final CheckBox formatInsteadCheckbox = viewRoot.findViewById(R.id.get_format_instead_date_or_time_checkbox);
-        final CheckBox alwaysNowCheckBox = viewRoot.findViewById(R.id.always_use_current_datetime_checkbox);
```

Comment:
Removed `alwaysNowCheckBox`. This isn't really useful. A 'set now' button might be better.